back-end/app.js

```diff
@@ -174,7 +175,11 @@ app.get('/shows-trakt', (req, res, next) => {
         res.json(response.data)
       })
       .catch((err) => {
-        next(err)
+        if (err.response.status === 500) {
+          res.status(200).json(mockPopularShows)
```

Comment:
So, instead of returning mock data, I'd suggest something more like `res.status(err.response.status).json("an error occurred loading shows")` and changing the corresponding test to see whether the body of the response matches "an error occurred loading shows" (or whatever error message we have the API return).